packages/rest-crud/README.md

```diff
@@ -15,11 +15,56 @@ npm install --save @loopback/rest-crud
 
 ## Basic use
 
-`@loopback/rest-crud` exposes two helper methods (`defineCrudRestController` and
-`defineCrudRepositoryClass`) for creating controllers and respositories using
-code.
+`@loopback/rest-crud` can be used along with `@loopback/model-api-builder` to
+easily create a repository class and a controller class for your model.
 
 For the examples in the following sections, we are assuming a model named
+`Product` and a datasource named `db` have already been created.
+
+In your `src/application.ts` file:
+
+```ts
+// add the following import
+import {CrudRestComponent} from '@loopback/rest-crud';
+
+export class TryApplication extends BootMixin(
+  ServiceMixin(RepositoryMixin(RestApplication)),
+) {
+  constructor(options: ApplicationConfig = {}) {
+    // other code
+
+    // add the following line
+    this.component(CrudRestComponent);
+  }
+}
+```
+
+Create a new file for the configuration, e.g.
+`src/model-endpoints/product.rest-config.ts` that defines the `model`,
+`pattern`, `dataSource`, and `basePath` properties:
+
+```ts
+import {ModelCrudRestApiConfig} from '@loopback/rest-crud';
+import {Product} from '../models';
+
+module.exports = <ModelCrudRestApiConfig>{
+  model: Product,
+  pattern: 'CrudRest',
+  dataSource: 'db',
+  basePath: '/products',
+};
+```
+
+Now your `Product` model will have a default repository and default controller
```

Comment:
So there are 2 ways that a developer can create the `default` controller and `default `repository for Product?
  1. Performing Basic steps  and **not** performing Advanced steps
  1. Performing Advanced steps and **not** performing Basic steps

So they are mutually exclusive then? If so let's specify that Basic and Advanced are mutually exclusive. Another question: why would someone want to do Advanced steps if it is shorter to perform Basic steps?

Reply:
> So they are mutually exclusive then? If so let's specify that Basic and Advanced are mutually exclusive.

Yeah, I'll specify that, thanks!

> Another question: why would someone want to do Advanced steps if it is shorter to perform Basic steps?

The advanced one helps if you only want to create the default controller or default repository but not both.